specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/secrets.json

```diff
@@ -573,95 +661,6 @@
         }
       }
     },
-    "/secrets/{secret-name}/rotationpolicy": {
```

Comment:
Why was this removed? Was it only so you could update the secret bundle with a policy? As noted above, that is going to create problems for the SDKs that have long since GA'd. I don't think all languages will be able to work around the issue.

I recommend keeping this, which would be consistent with Keys.

Reply:
Get/Set key rotation policy APIs were created to work with MHSM.  However, we don't have such dependencies in secret. 
Another reason that we want to combine the get/set rotation policy with the secret get/path operation is to make it work better with Azure Policy.  @jlichwa can add more details here :)

Reply:
Are you saying MHSM does support this endpoint but KV doesn't? If that is the case, the endpoint needs to stay but perhaps KV can return an error - 501 not implemented - while MHSM supports it.

Reply:
AKV and MHSM support key rotation policy, but only AKV supports secrets.

/cc @dhruv-iyer